gatsby-browser.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Browser APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/browser-apis/
+ */
+
+// You can delete this file if you're not using it
```

Comment:
Q: are we using it?

Reply:
Yeah I think it's safe to delete this now, we are not going to use it any time soon here.